aws/data_source_aws_cloudwatch_log_group_test.go

```diff
@@ -0,0 +1,36 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSCloudwatchLogGroupDataSource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckAWSCloudwatchLogGroupDataSourceConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.aws_cloudwatch_log_group.test", "name", "Test"),
+					resource.TestCheckResourceAttrSet("data.aws_cloudwatch_log_group.test", "arn"),
+					resource.TestCheckResourceAttrSet("data.aws_cloudwatch_log_group.test", "creation_time"),
+				),
+			},
+		},
+	})
+	return
+}
+
+var testAccCheckAWSCloudwatchLogGroupDataSourceConfig = fmt.Sprintf(`
+resource aws_cloudwatch_log_group "test" {
+  name = "Test"
```

Comment:
We should randomize the naming here in case of AWS/testing failures 😄  I will fix this during merge to be `%s`, make `testAccCheckAWSCloudwatchLogGroupDataSourceConfig` a function with `rName string` parameter, and generate the random name via something like `rName := acctest.RandomWithPrefix("tf-acc-test")`